remotecv/detectors/feature_detector/__init__.py

```diff
@@ -32,6 +35,12 @@ def detect(self, image):
                 left, top = corner.ravel()
                 points.append([left, top, 1, 1])
 
+            context.metrics.timing(
```

Comment:
@devppjr can you address the codeclimate issue?

Reply:
You could wrap the detect calls in a `with` statement to reduce the duplication.

Reply:
@guilhermef What about now?